src/lib/blockchains/nxt.coffee

```diff
@@ -75,7 +75,7 @@ module.exports = (dnschain) ->
                     res.on 'end', () ->
                         try
                             response = JSON.parse data
-                            result.data = JSON.parse response.aliasURI
+                            result.data.value = JSON.parse response.aliasURI
```

Comment:
@toenu23 It's OK, I'm going to add some tests for NXT support as well, and in the process i'll make sure this works.